tools/codegen/api/types.py

```diff
@@ -40,15 +53,221 @@ def __str__(self) -> str:
             mb_default = f"={self.default}"
         return f"{self.type} {self.name}{mb_default}"
 
+    # List of CppArguments that this structure explicitly represents
+    def explicit_arguments(self) -> Sequence['CppArgument']:
```

Comment:
you don't need this anymore, right? `CppArgument` is the low level data structure that was already unpacked.